src/components/EditorBottomBar.tsx

```diff
@@ -1,15 +1,20 @@
 import "./EditorBottomBar.css";
+import { useIntl } from "react-intl";
 
 interface EditorBottomBarProps {}
 
 export const EditorBottomBar = ({ ...otherProps }: EditorBottomBarProps) => {
+  const intl = useIntl();
+  const translate = (id?: any, values?: any) =>
+    intl.formatMessage({ id: id }, values);
```

Comment:
Esto funcionará? Igual dejemoslo para el PR de refactorización

```suggestion
  const _ = useIntl().formatMessage;
```